spec/services/request_password_reset_spec.rb

```diff
@@ -1,6 +1,8 @@
 require 'rails_helper'
 
 describe RequestPasswordReset do
+  let(:analytics) { FakeAnalytics.new }
```

Comment:
Is this doing anything?

Do we not have any coverage for throttled flows in these specs?

Reply:
It is not but it should be since this was a note of sorts to myself to add a test. Added a test in 3d1780d6e13a4d2eb06de610b9ed898127e0dbc1